suite-native/assets/src/components/AssetItem.tsx

```diff
@@ -62,18 +50,9 @@ export const AssetItem = ({
                         alignItems="center"
                     >
                         <Text>{cryptoCurrencyName}</Text>
-                        <Text>{FiatAmountFormatter.format(fiatCurrencyValue)}</Text>
+                        <Text>{FiatAmountFormatter.format(fiatBalance)}</Text>
```

Comment:
How is possible that you don't need to currency as param here?

Reply:
because of the provider that has been refactored in https://github.com/trezor/trezor-suite/pull/6530